package.json

```diff
@@ -41,7 +41,7 @@
     "mime-types": "^2.1.13",
     "pane-registry": ">=1.0.0",
     "rdflib": ">=0.17.0",
-    "solid-ui": ">=0.11.3"
+    "solid-ui": ">=0.12.8-0"
```

Comment:
You should not need to do this, as it has already been done in 9cfe9b41d559305d334891f00fce2dc05889082b . Please rebase your branch on `master`.